tests/isolated/regular_file_readall.py

```diff
@@ -0,0 +1,42 @@
+import eventlet
+eventlet.monkey_patch()
```

Comment:
thanks, hadn't noticed that the standards had moved since that test was written. updated now